src/stack_manager.py

```diff
@@ -80,35 +88,28 @@ def parse(self, tag_list: List[List]) -> List:
 
         self.lookaheads = 0
         for tag, *scoped_tags in tag_list:
-            if is_layout_obj(tag):  # create new scope for layout object
```

Comment:
`is_layout_obj` is no longer used. Should we delete it?

Reply:
done